benchmarks/differenciation/python/aspect_python.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
```

Comment:
Is the content of this files also in #3088? Is there a way to use only one (when both are merged)?

Reply:
There would be a way, by packaging the python code into a pip package. However, this seems to be a little bit extreme for a couple of functions. So far, it is the same file than in the PR #3088, and using links to it is a little bit weird. I could remove it here and ask people to copy paste the file here...
for now, I think it's simpler to use a file with the functions here (especially as there is no tests on the python code, so even with a python package, there would be no way to be sure that nothing has been broken later and that these functions no longer work)